ceph/ceph_admin/upgrade.py

```diff
@@ -65,7 +66,19 @@ def upgrade_status(self: OrchProtocol):
 
         """
         out, _ = self.shell(args=["ceph", "orch", "upgrade", "status"])
-        return loads(out)
+        try:
+            return loads(out)
+        except JSONDecodeError:
+            pass  # Not valid JSON, move to next check
+
+        if "There are no upgrades in progress currently." in out:
+            return out
```

Comment:
Based on this result, [1] needs to be changed.
[1] https://github.com/red-hat-storage/cephci/pull/4699/files#diff-1b007be31b8ae465e57cdc902a3875d9d343cd2f7dba22fce8c222ccd361c534R112